xwiki-release-scripts/maven-release.sh

```diff
@@ -104,7 +104,7 @@ function check_versions() {
     exit -1
   fi
 
-  if [ $do_release ]
+  if [ "$do_release" = true ]
```

Comment:
Overall, I have a doubt if a for with double square brackets should be used instead.

```
if [[ "$do_release" = true ]]
```